### ISSUE TITLE
healthCheckCommand.String() is slightly confusing

### ISSUE THREAD

**Issue by bboreham** [kubernetes/kubernetes#29107]
E.g. I get a message like this:

````
  Unable to start node services.
  Timeout waiting for service `/home/vagrant/bin/etcd etcd` health-check: http://127.0.0.1:4001/v2/keys/
````

it looks like it would run the command `/home/vagrant/bin/etcd etcd`.  But this is misleading: the second `etcd` is just the `argv[0]` that it passes on invocation.

**Comment by apelisse** [comment]
cc @pwittrock @timothysc

**Comment by pwittrock** [comment]
Will be fixed in #29110
